hello_world_print.py

```diff
@@ -1,3 +1,15 @@
 # A program to print hello world using the print function
 
 print('Hello, World!')
+
+#program to print hello world using fstring
+a='world'
+print(f"hello {a}")
+
+#print hello world multiple times using while loop
+
+i=1
+print("\nprinting hello world multiple times\n")
+while i<=10:
+    print("hello world")
+    i+=1
```

Comment:
A while loop implementation has already been done in the repository so I won't be accepting redundant solution again.